assets/tests/components/routeLadder.test.tsx

```diff
@@ -225,6 +225,39 @@ describe("routeLadder", () => {
     expect(tree).toMatchSnapshot()
   })
 
+  test("renders a route ladder with the new header and detour dropdown", () => {
+    jest
+      .mocked(getTestGroups)
+      .mockReturnValue(["route-ladder-header-update", "detours-pilot"])
+
+    const route: Route = routeFactory.build({
+      id: "28",
+      name: "28",
+    })
+    const timepoints = [
+      { id: "MATPN", name: "MATPN Name" },
+      { id: "WELLH", name: "WELLH Name" },
+      { id: "MORTN", name: "MORTN Name" },
+    ]
+
+    const tree = render(
+      <RouteLadder
+        route={route}
+        timepoints={timepoints}
+        vehiclesAndGhosts={undefined}
+        selectedVehicleId={undefined}
+        deselectRoute={() => {}}
+        reverseLadder={() => {}}
+        toggleCrowding={() => {}}
+        ladderDirections={{}}
+        ladderCrowdingToggles={{}}
+        hasAlert={false}
+      />
+    ).container
```

Comment:
suggestion: could you de-structure instead? (I see other tests did it this way, and I think they're wrong )
```suggestion
    const { container: tree } = render(
      <RouteLadder
        route={route}
        timepoints={timepoints}
        vehiclesAndGhosts={undefined}
        selectedVehicleId={undefined}
        deselectRoute={() => {}}
        reverseLadder={() => {}}
        toggleCrowding={() => {}}
        ladderDirections={{}}
        ladderCrowdingToggles={{}}
        hasAlert={false}
      />
    )
```